bin/backend_check

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+"""
+This script checks that all terraform directories under the directory provided contain backend configs.
+
+Usage:
+    backend_check [options] [PATH]
+
+Arguments:
+    PATH    The path of the directory to check. Defaults to the current working directory.
+
+Options:
+    -h, --help          Show this message and exit.
+    --version           Show the version.
+"""
+import concurrent.futures
+import os
+import sys
+from typing import List
+
+import hcl
+from docopt import docopt
+
+from terrawrap.version import __version__
+
+
+def main():
+    arguments = docopt(__doc__, version=__version__)
+    target_dir = arguments["PATH"] if arguments["PATH"] else os.getcwd()
+
+    config_dirs = {
```

Comment:
Nitpick: We probably should make a utility function for this later since we are doing this pretty often

Reply:
I think this is the second time, so we can refactor it if we need it again.